packages/block-editor/src/hooks/background.js

```diff
@@ -420,7 +429,7 @@ function BackgroundSizePanelItem( {
 			sizeValue !== 'contain' ) ||
 		sizeValue === ''
 			? 'auto'
-			: sizeValue || 'cover';
+			: sizeValue || defaultValue;
```

Comment:
It looks like this changes the behaviour in a subtle way, as an `undefined` value will now be treated as whatever `defaultValue` is, which no longer defaults to `cover`. So we might want to update the comment above this `const`?